packages/shared/styles/components/SfAccordion.scss

```diff
@@ -0,0 +1,51 @@
+@import '../variables';
+
+$sf-accordion-item__header-padding: 1rem 2rem;
```

Comment:
```suggestion
$accordion-item__header-padding: 1rem 2rem !default;
```
SCSS vars shouldn't have `$sf-` prefix, and you should use `!default` for all component variables

Reply:
same for all vars below...

Reply:
Thanks, updated :-)